lms/templates/api_admin/api_access_request_email_approved.txt

```diff
@@ -0,0 +1,21 @@
+## mako
+Dear ${name},
+
+Your request for access to the ${platform_name} Course Catalog API has been approved. Complete the following steps to create your API credentials:
+
+  1. Sign in to ${api_management_url} using the same account you used to request API access.
```

Comment:
Either our UI should be intuitive enough to understand without instructions, or the instructions should be on the page itself. Adding instructions to the email is weird. Once I click the link, I should not need to refer back to the email to determine how to continue on the website.

The only thing this email should say, is "click here to get setup".